common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -103,12 +109,49 @@ like_elekks_trigger = {
 		has_innovation = innovation_elephantry
 	}
 }
+like_dragonhawk_trigger = {
+	OR = {
```

Comment:
You shouldn't forget about oasis terrain. Centaurs and other units don't have bonuses there.

Reply:
Hm, I guess oasises could stay out of it. Others not used to desert/dryland fighting might still be ok at an oasis, removing the need for a oasis bonus.

Reply:
> Hm, I guess oasises could stay out of it. Others not used to desert/dryland fighting might still be ok at an oasis, removing the need for a oasis bonus.

Camels fight good in oasis. Paradox see oasis as a better variant of desert or dryland.

Reply:
Do you mean for all light cav, or for all centaurs?

Reply:
My point short: all units that live in deserts should have oasis bonus as well as desert bonus.

Reply:
Did not add for deathcaller, impaler (they already have steppe bonus). Choasborn and impaler didn't add as well. I added where I could but I don't think this terrain should be as emphasized as others...it is really not used as much, and putting too many units points into terrain, especially terrain that is rare, makes the units not even comparable in quality to other units, that are less dependant on terrain. It basically makes them strong only on home territory, and I think doing this will give the effect where we will see a lot less conquering being done.

Reply:
> and putting too many units points into terrain, especially terrain that is rare, makes the units not even comparable in quality to other units

Reduce the cost of these bonuses.

> Did not add for deathcaller, impaler (they already have steppe bonus).

Yeah, about this. Centaurs live near oasises in the Barrens.